common-tools/clas-detector/src/main/java/org/jlab/detector/calib/utils/ConstantsManager.java

```diff
@@ -138,7 +138,7 @@ else if (requests > 1) {
             try {
                 IndexedTable  table = provider.readTable(tableName, desc.getTableIndices().get(i));
                 desc.getMap().put(tk.get(i), table);
-                LOGGER.log(Level.INFO, String.format("***** >>> adding : %14s / table = %s", tk.get(i), tableName));
+                LOGGER.log(Level.FINE, String.format("***** >>> adding : %14s / table = %s", tk.get(i), tableName));
```

Comment:
Switching to FINE seems to suppress the CCDB printouts when running recon-util. Is this intended?